library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -74,26 +73,41 @@ PlatformBridgeFilter::PlatformBridgeFilter(PlatformBridgeFilterConfigSharedPtr c
   if (platform_filter_.init_filter == nullptr) {
     ENVOY_LOG(debug, "PlatformBridgeFilter({}): missing initializer", filter_name_);
     platform_filter_ = {};
+
+    // Does this make sense, or should we rather protect the rest of the code against nullptr for
+    // both request_filter_base_ and response_filter_base_.
```

Comment:
This makes sense, but I'd switch to an `if`/`else` and invert the check so the "happy" path comes first, that way the constructor can terminate with setting the two bases in either case.